packages/palette/src/svgs/EarnMoreIcon.tsx

```diff
@@ -0,0 +1,56 @@
+import React from "react"
+import { color } from "../helpers"
+import { Icon, IconProps, Title } from "./Icon"
+
+/** EarnMoreIcon **/
+export const EarnMoreIcon: React.FC<IconProps> = ({
+  title = "Earn More",
+  ...props
+}) => {
+  return (
+    <Icon
+      {...props}
+      // viewBox="0 0 18 18"
+      xmlns="http://www.w3.org/2000/svg"
+      // width="56"
+      // height="75"
```

Comment:
Can we remove these commented out props? 

One helpful tip before opening a PR (or after opening, but before assigning to another dev) is to use the `Files changed` tab on the GitHub UI and do a careful self-review; this will often catch things like stray `console.log`'s, commented out code and so on, but more importantly looking at the code one writes in a new context away from VSCode will often reveal blindspots or other refactor possibilities.

Reply:
Thanks for noticing, my eyes didn't catch these!